core/math/transform_interpolator.cpp

```diff
@@ -30,6 +30,51 @@
 
 #include "transform_interpolator.h"
 
+#include "core/math/transform_2d.h"
+
+void TransformInterpolator::interpolate_transform2D(const Transform2D &p_prev, const Transform2D &p_curr, Transform2D &r_result, real_t p_fraction) {
```

Comment:
Should be snake case:
```suggestion
void TransformInterpolator::interpolate_transform_2d(const Transform2D &p_prev, const Transform2D &p_curr, Transform2D &r_result, real_t p_fraction) {
```